[Document] Introduces the document of the command line interface (#204)
flink-extended/ai-flow: docs/content/cli/aiflow.md

# AIFlow

## Command Line Interface

> AIFlow has a very rich command line interface that allows for many types of operation on a Workflow, starting services, and supporting development and testing.

Reviewer: ```suggestion
AIFlow has a very rich command-line interface that supports many types of operations on a Workflow, starting services and testing.
```

```
aiflow job show-execution [-h] [-o table, json, yaml, plain] project_path job_name workflow_execution_id
```

##### Positional Arguments

> project_path

The path of the project.

> job_name

The name of the job.

> workflow_execution_id

The id of the workflow execution.

##### Named Arguments

> -o, --output

Possible choices: table, json, yaml, plain.  
Output format. Allowed values: json, yaml, plain, table (default: table).  
Default: "table".

##### start-execution

Starts the job execution by job name and workflow execution id.

```
aiflow job start-execution [-h] project_path job_name workflow_execution_id 
```

##### Positional Arguments

> project_path

The path of the project.

> job_name

The name of the job.

> workflow_execution_id

The id of the workflow execution.

##### stop-execution

Stops the job execution by job name and workflow execution id.

```
aiflow job stop-execution [-h] project_path job_name workflow_execution_id
```

##### Positional Arguments

> project_path

The path of the project.

> job_name

The name of the job.

> workflow_execution_id

The id of the workflow execution.

(aiflow-cli-config)=

#### config

> Manage configuration.

Reviewer: ```suggestion
Manages configuration.
```

```
aiflow config [-h] COMMAND ... 
```

#### Positional Arguments

> COMMAND

Possible choices: get-value, init, list.

#### Sub-commands

##### get-value

Gets the option value of the configuration.

```
aiflow config get-value [-h] option
```
 
##### Positional Arguments

> option

The option name of the configuration.

##### init

Initializes the default configuration.

```
aiflow config init [-h]
```

##### list

> List all options of the configuration.

Reviewer: ```suggestion
Lists all options of the configuration.
```
